src/hooks/useViewportColumns.ts

```diff
@@ -16,21 +15,27 @@ interface ViewportColumnsArgs<R, K extends keyof R, SR> extends SharedDataGridPr
 
 export function useViewportColumns<R, K extends keyof R, SR>({
   columns: rawColumns,
-  minColumnWidth,
   columnWidths,
   viewportWidth,
-  defaultFormatter,
-  scrollLeft
+  scrollLeft,
+  defaultColumnOptions
 }: ViewportColumnsArgs<R, K, SR>) {
+  const minColumnWidth = defaultColumnOptions?.minWidth ?? 80;
+  const defaultFormatter = defaultColumnOptions?.formatter ?? ValueFormatter;
+  const defaultSortable = defaultColumnOptions?.sortable;
+  const defaultResizable = defaultColumnOptions?.resizable;
```

Comment:
Let's enforce non-nullish types when we can. We can simplify the calculated column type as well that way I think.
```suggestion
  const defaultSortable = defaultColumnOptions?.sortable ?? false;
  const defaultResizable = defaultColumnOptions?.resizable ?? false;
```